mcodingbot/plugins/python.py

```diff
@@ -2,15 +2,16 @@
 
 import crescent
 import hikari
+from crescent.ext import tasks
 
-from mcodingbot.utils import Plugin
+from mcodingbot.config import CONFIG
+from mcodingbot.utils import PepManager, Plugin
 
 plugin = Plugin()
 
 PEP_REGEX = re.compile(r"pep[\s-]*(?P<pep>\d{1,4}\b)", re.IGNORECASE)
-
-
 DISMISS_BUTTON_ID = "dismiss"
+PEP_MANAGER = PepManager()
```

Comment:
so the question is, should this actually be a constant, or just a global?

Usually constants are constant, but this pep manager is modifiable (and it is modified, every hour). Should this maybe be all lower case, and just go under plugin?

Reply:
`SCREAMING_SNAKE_CASE` is more clearly a global so I would stick with it.
Otherwise put it back in the plugin class I guess.

Reply:
@trag1c what do you think